CodeSniffer/CLI.php

```diff
@@ -1276,4 +1284,23 @@ private function _validateReportWidth($width)
     }//end _validateReportWidth()
 
 
+    /**
+     * Checks if the given command exists on the system
+     *
+     * @param string $cmd The name of the command
+     *
+     * @return bool
+     */
+    private function _commandExist($cmd)
+    {
+        $returnVal = shell_exec("which $cmd  2> /dev/null");
```

Comment:
Ha, now `which` only available on Unix-like systems. Instead you should use OS detection and for Windows use `where` command (see http://stackoverflow.com/questions/304319/is-there-an-equivalent-of-which-on-the-windows-command-line).